official/boosted_trees/README.md

```diff
@@ -18,7 +18,7 @@ First make sure you've [added the models folder to your Python path](/official/#
 The [HIGGS Data Set](https://archive.ics.uci.edu/ml/datasets/HIGGS) that this sample uses for training is hosted by the [UC Irvine Machine Learning Repository](https://archive.ics.uci.edu/ml/datasets/). We have provided a script that downloads and cleans the necessary files.
 
 ```
-python data_download.py
+python -m official.boosted_trees.data_download  # in the github cloned directory
```

Comment:
As the `path_to/model ` is required to be added to python path, I think it's not necessary to add the full path to the py file. Here is the link for more details: https://github.com/tensorflow/models/tree/master/official#running-the-models

Reply:
I find that that "adding model path to $PATH" is not needed when running at the specified directory, and using -m with module structure (official.boosted_trees.data_download), so did this.

However, as it's tested only in my environment, it might not be compatible to others.
Reverted back.